docs/en/siem/siem-ui.asciidoc

```diff
@@ -51,12 +51,13 @@ Interactive widgets let you drill down for deeper insights:
 
 There are also tabs for viewing and investigating specific types of data:
 
-* All hosts
-* Authentications
-* Uncommon processes
-* Anomalies
-* Events
-* <<det-engine-terminology, External alerts>>
+* All hosts: high-level host details
+* Authentications: authentication events
+* Uncommon processes: uncommon processes running on hosts
+* Anomalies: anomalies discovered by <<machine-learning, machine learning jobs>>
+* Events: all host events
+* <<det-engine-terminology, External alerts>>: alerts received from 
+external monitoring tools, such as Suricata
```

Comment:
Can we change this one to include Elastic Endpoint Security?  Perhaps "external security tools, such as Elastic Endpoint Security"